### PR TITLE
Fix background not loading in Orange Wall and Text themes

### LINKED ISSUE #611: Text theme can't load background
In #545, 'Tex' got renamed to 'Text' (as it should have been), but the [URL for the background image](https://github.com/new-xkit/XKit/commit/0d5901e3ae0553f7a157b59d5dbd64872d4650b4#diff-0fbf2a8d2c4b2c178733b3318a0a8b67R7) was not updated to reflect this change.

### PR DESCRIPTION
**Pull request by finagle29**
Fixes #611

### CODE REVIEW

File: Themes/text/text.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* NAME Text **/
-/* VERSION 1.0.2 **/
+/* VERSION 1.0.3 **/
 /* DESCRIPTION Slightly modified Default theme **/
 /* DEVELOPER STUDIOXENIX **/
 
 .identity {
-	background: #3c5a76 url('https://new-xkit.github.io/XKit/Themes/tex/body-background.jpg') no-repeat center top !important;
+	background: #3c5a76 url('https://new-xkit.github.io/XKit/Themes/text/body-background.jpg') no-repeat center top !important;
```
Comment (Wolvan) [inline Themes/text/text.css]: tried opening that link in my browser, 404

Comment (0xazure) [inline Themes/text/text.css]: Should be `.png` based on https://github.com/new-xkit/XKit/blob/master/Themes/text/body-background.png

Comment (finagle29) [inline Themes/text/text.css]: thanks for catching that

**Comment by Wolvan** [comment]
LGTM, @homu

**Comment by Wolvan** [comment]
LGTM, @homu r+

**Pull request merged**